src/platform/tls_schannel.c

```diff
@@ -2222,6 +2223,17 @@ QuicTlsProcessData(
         goto Error;
     }
 
+    if (!TlsContext->IsServer && State->BufferOffset1Rtt > 0 &&
+        State->ReadKeys[QUIC_PACKET_KEY_HANDSHAKE] == NULL) {
```

Comment:
Why are you checking the ReadKey?

Reply:
Ideally, I'd like to read `Connection->State.HandshakeConfirmed` but that's not exposed to the TLS layer.
So I checked what changes in the TLS state when `QuicCryptoOnHandshakeConfirmed` is called, and the ReadKeys for Handshake are set to NULL.

Reply:
miTLS code doesn't wait for handshake confirmed and it passes. Why are you trying to impose that restriction here?

Reply:
miTLS also gets a notification when the ticket is received. Since Schannel doesn't provide that interface we have to guess at a milestone where we can be certain the ticket has been received.

Reply:
At least for now, the only 1-RTT crypto data exchanged is NST. So when a client receives 1-RTT crypto data is MUST be a ticket. Also, miTLS always indicates inline when we get the ticket. So, removing this additional check is equivalent to what miTLS does already. When we do get an indication from Schannel I expect it to be in response to a call to them and also be equivalent to this.

Reply:
When we get notifications from Schannel this check will go away and things will work appropriately. This is just a temporary workaround.